extensions/wikia/TemplateClassification/TemplateClassificationController.class.php

```diff
@@ -23,6 +23,7 @@ public function getTemplateClassificationEditForm() {
 				 * template-classification-type-unclassified
 				 */
 				'name' => wfMessage( "template-classification-type-{$type}" )->escaped(),
+				'description' => wfMessage( "template-classification-description-{$type}" )->escaped(),
```

Comment:
Since the message key is being constructed with a variable, similar to the message use above, this should have a comment above it with the possible message strings that can be used here so it is possible to find where they are used with grep.